### PR TITLE
Add mp4v2 package

### PR DESCRIPTION
**Pull request by uberhacker**
MP4v2 Library: This library provides functions to read, create, and modify mp4 files.  See https://code.google.com/archive/p/mp4v2/.

### CODE REVIEW

File: packages/mp4v2.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+class Mp4v2 < Package
+  description 'MP4v2 Library: This library provides functions to read, create, and modify mp4 files'
+  homepage 'https://code.google.com/archive/p/mp4v2/'
+  version '2.0.0'
+  source_url 'https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/mp4v2/mp4v2-2.0.0.tar.bz2'
+  source_sha256 '0319b9a60b667cf10ee0ec7505eb7bdc0a2e21ca7a93db96ec5bd758e3428338'
+
+  def self.build
+    system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
```
Comment (cstrouse) [inline packages/mp4v2.rb]: Does it make sense to add `--disable-dependency-tracking` here to speed up the build? And maybe also `--disable-debug`?

Comment (uberhacker) [reply packages/mp4v2.rb]: Those appear to be for development purposes only, so it definitely makes sense.  Good find.

**Comment by cstrouse** [comment]
Works on Samsung Chromebook 3 (x86_64) but fails on Samsung Chromebook Plus (aarch64).

```bash
chronos@localhost /usr/local/lib/crew/packages $ crew install mp4v2 -s -k
mp4v2: MP4v2 Library: This library provides functions to read, create, and modify mp4 files
https://code.google.com/archive/p/mp4v2/
version 2.0.0
Downloading source...
######################################################################## 100.0%
Archive downloaded
Unpacking archive, this may take awhile...
gcc (GCC) 4.9.4
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Building from source, this may take a while...
configure:
  -->
  --> Configuring MP4v2 2.0.0
  -->
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking build system type... autoaux/config.guess: line 1454: /tmp/cgwbCiOY/dummy: Permission denied
autoaux/config.guess: unable to guess system type

This script, last modified 2009-06-10, has failed to recognize
the operating system you are using. It is advised that you
download the most up to date version of the config scripts from

  http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD
and
  http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD

If the version you run (autoaux/config.guess) is already up to date, please
send the following data and any information you think might be
pertinent to <config-patches@gnu.org> in order to provide the needed
information to handle your system.

config.guess timestamp = 2009-06-10

uname -m = aarch64
uname -r = 4.4.70-11386-g8271523ecf43
uname -s = Linux
uname -v = #1 SMP PREEMPT Sun Oct 15 17:38:26 PDT 2017

/usr/bin/uname -p = ARMv8 Processor rev 4 (v8l)
/bin/uname -X     = 

hostinfo               = 
/bin/universe          = 
/usr/bin/arch -k       = 
/bin/arch              = 
/usr/bin/oslevel       = 
/usr/convex/getsysinfo = 

UNAME_MACHINE = aarch64
UNAME_RELEASE = 4.4.70-11386-g8271523ecf43
UNAME_SYSTEM  = Linux
UNAME_VERSION = #1 SMP PREEMPT Sun Oct 15 17:38:26 PDT 2017
configure: error: cannot guess build type; you must specify one
mp4v2 failed to install: `./configure             --prefix=/usr/local             --libdir=/usr/local/lib             --disable-dependency-tracking             --disable-debug` exited with 1
```

**Comment by cstrouse** [comment]
For it to compile properly on aarch64 you need to add the following configure options:

```
--host=aarch64-unknown-linux-gnu \
--build=aarch64-unknown-linux-gnu
```

**Comment by uberhacker** [comment]
Replaced by #1373.

**Pull request closed without merging**